components/monday/sources/common/common-webhook.mjs

```diff
@@ -16,18 +16,6 @@ export default {
     },
   },
   hooks: {
```

Comment:
I think it would make sense to put the deploy hook in `common-webhook` and change `commonDeploy()` to `emitHistoricalEvents()`.  Then, for sources that don't use that method, you can overwrite it with another `emitHistoricalEvents()` method to list that component's historical events.
```suggestion
  hooks: {
     async deploy() {
        await this.emitHistoricalEvents();
     },
```